README.md

```diff
@@ -1,58 +1,45 @@
-# Open棟梁(OpenTouryo)
-”Open棟梁”は、長年の.NETアプリケーション開発実績にて蓄積したノウハウに基づき開発した.NET用アプリケーション フレームワークです。
-(”OpenTouryo” , is an application framework for .NET which was developed using the accumulated know-how with a long track record in .NET application development.)
+# OpenTouryo
+*Open Touryo* is an application framework for .NET Framework which was developed using the accumulated know-how with a longstanding application development results.
```

Comment:
@daisukenishino 

Please add link to Japanese version.
See [readme file in OpenTouryoTempates repository](https://github.com/OpenTouryoProject/OpenTouryoTemplates/blob/master/root_VS2015/Readme.md).
```md
Click [here](Readme.ja.md) for Japanese version of this file.
```